.github/workflows/check_api_docs.yml

```diff
@@ -21,33 +28,34 @@ on:
           - organisations
           - personal-relationships
           - sentencing-adjustments
-
-permissions:
-  contents: write
-  pull-requests: write
-
-jobs:
-  build:
-    runs-on: [ self-hosted, hmpps-github-actions-runner ]
     steps:
       - uses: actions/checkout@v4
 
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin' # See 'Supported distributions' for available options
+          java-version: '21'
+          cache: 'gradle'
+
       - name: Check API docs
         id: check-api
-        run: ./check-api-docs.bash ${{ inputs.project }}
+        run: ./check-api-docs.bash ${{ matrix.project }}
 
       - name: Upload differences report
-        if: check-api.outputs.differences == 'true'
+        if: steps.check-api.outputs.differences == 'true'
         uses: actions/upload-artifact@v4
         with:
           path: build/api.diff
 
       - name: Create Pull Request
-        if: check-api.outputs.differences == 'true'
+        if: steps.check-api.outputs.differences == 'true'
         uses: peter-evans/create-pull-request@v7
         with:
-          commit-message: Update api docs to ${{ check-api.output.new_version }}
-          title: Update api docs to ${{ check-api.output.new_version }}
-          body: Update api docs from ${{ check-api.output.old_version }} to from ${{ check-api.output.new_version }}
-          branch: api-docs-${{ inputs.project }}
+          commit-message: Update api docs for ${{ matrix.project }} to ${{ steps.check-api.outputs.new_version }}
+          title: Update api docs for ${{ matrix.project }} to ${{ steps.check-api.outputs.new_version }}
+          body: |
+            Update api docs from ${{ steps.check-api.outputs.old_version }} to ${{ steps.check-api.outputs.new_version }}
+
+            Current production version is ${{ steps.check-api.outputs.production_version }}
+          branch: api-docs-${{ matrix.project }}
```

Comment:
Would this clash if a previous PR wasn't merged/closed and fail to create this PR (because branch already exists)?